### PR TITLE
Make "custom activity" in the readme real PHP

### PR DESCRIPTION
**Pull request by szepeviktor**
It was half pseudo code.

### CODE REVIEW

**Comment by szepeviktor** [comment]
@JJJ Should I include the markdown version? Or is it auto generated?

File: readme.txt [hunk 1]
```diff
@@ -82,8 +82,9 @@ Yes. It will work perfectly with all post-types & taxonomies that define their o
 Yes. The autoloader can be filtered, so adding new object types is as simple as:
 
 `
-add_filter( 'wp_get_default_user_activity_types', function( $types = array() ) {
-	$types[] = 'Your_New_Type' // class that extends WP_User_Activity_Type
+add_filter( 'wp_get_default_user_activity_types', function ( $types ) {
```
Comment (JJJ) [inline readme.txt:85]: `$types` needs to stay an array here, otherwise this looks good.

Comment (szepeviktor) [reply readme.txt:85]: I think it is always an array https://github.com/stuttter/wp-user-activity/blob/686bc51cf6524968fe20b20cba9bbd4af8e78f89/wp-user-activity/includes/common.php#L31

Comment (JJJ) [reply readme.txt:85]: It always starts that way, but the `wp_get_default_user_activity_types` filter means someone could return `false` or `null`.

**Pull request closed without merging**